server/src/main/java/org/elasticsearch/cluster/coordination/CoordinationMetaData.java

```diff
@@ -220,6 +228,97 @@ public CoordinationMetaData build() {
         }
     }
 
+    public static class VotingTombstone implements Writeable, ToXContentFragment {
+        private String nodeId;
```

Comment:
`final`?

Reply:
I hate that things are not final by default in Java. https://github.com/elastic/elasticsearch/pull/35832/commits/10373700702238f10dbc8e7dc6b69cc6c6b3ca7e